### DISCUSSION TITLE
[TCOTD2] Done with Thamuz

### DISCUSSION THREAD

**Discussion by Blue-Shadow** [Realm667/Re-Releases#272] [General]
Apart from tweaking numbers, like damage and the like, yeah, I think I'm done for now with the work on Thamuz. I've been doing my tests in Doom 2's MAP32, so I don't know how it'll turn out in the actual game.

If you have any feedback, let me hear it.

**Comment by Tormentor667** [comment]
For me it's good as it is :) Well done!
